lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -122,6 +133,18 @@
     db: acme
     cascade: yes
     state: absent
+
+- name: Create extension dummy of version 1.2 or update it if it's already created
```

Comment:
- name: Create dummy extension of version 1.2 or update it if it's already created

Reply:
dummy is a name of extension :)

Reply:
@pratikgadiya12 , thanks, fixed